UnityProject/Assets/Scripts/UI/Systems/PreRound/GUI_PreRoundWindow.cs

```diff
@@ -336,6 +344,10 @@ public void CloseMapLoadingPanel()
 			normalWindows.SetActive(true);
 			mapLoadingPanel.SetActive(false);
 			UpdateLoadingBar("Preparing..", 0.1f);
+			if (PlayerManager.CharacterManager.Characters.Any() == false)
```

Comment:
this also could include the countdown?

Reply:
wdym?

Reply:
what happens if you don't have a character settings and you click ready? during the countdown

Reply:
It just activates the root window

Reply:
ok cool